File: server/freeScoreBoard7/Core/DB/Spieler.cs
namespace FreeScoreBoard.Core.DB
{
    using System;
    using System.Collections.Generic;
    using System.ComponentModel.DataAnnotations;
    using System.ComponentModel.DataAnnotations.Schema;
    using System.Data.Entity.Spatial;

    [Table("Spieler")]
    public partial class Spieler
    {
        public long ID { get; set; }

        public long? MannschaftsID { get; set; }

        [StringLength(2147483647)]
        public string Nachname { get; set; }

        [StringLength(2147483647)]
        public string Vorname { get; set; }

        [StringLength(2147483647)]
        public string SID { get; set; }

        [StringLength(2147483647)]
        public string NickName { get; set; }

        [StringLength(2147483647)]
        public string Kurzname { get; set; }

        [StringLength(2147483647)]
        public string Bild { get; set; }
    }
}

File: control page/src/teams/dataClasses/Spieler.js
export default class Spieler {
  constructor(
    _ID,
    _MannschaftsID,
    _Nachname,
    _Vorname,
    _SID,
    _NickName,
    _Kurzname,
    _Bild,
  ) {
    this.ID = _ID;
    this.MannschaftsID = _MannschaftsID;
    this.Nachname = _Nachname;
    this.Vorname = _Vorname;
    this.SID = _SID;
    this.NickName = _NickName;
    this.Kurzname = _Kurzname;
    this.Bild = _Bild;
  }
}
